src/Common/dataAccess/createDocument.ts

```diff
@@ -1,8 +1,8 @@
 import { CollectionBase } from "../../Contracts/ViewModels";
+import { logConsoleInfo, logConsoleProgress } from "../../Utils/NotificationConsoleUtils";
```

Comment:
i think my auto formatter from this package's modules was changing the order of these. Is this supposed to happen?

Reply:
yes